src/solver/application/application.cpp

```diff
@@ -229,7 +229,10 @@ void Application::readDataForTheStudy(Data::StudyLoadOptions& options)
     ScenarioBuilderOwner(study).callScenarioBuilder();
 }
 
-void Application::startSimulation(Data::StudyLoadOptions& options)
+// TODO : this function is too long and has a bad name.
+// TODO : we should split it into (at least) 4 functions.
+// TODO : Naming will be easier.
+void Application::readStudy_makeChecks_and_printThings(Data::StudyLoadOptions& options)
```

Comment:
Quickly rename this function with a less (and imperfect) misleading name.
Previous name was not good at all. This one is long but more accurate.